source/community/reactnative/src/components/containers/column.js

```diff
@@ -174,7 +174,7 @@ export class Column extends React.Component {
 		const minHeight = Utils.convertStringToNumber(this.column.minHeight);
 		//We will pass the style as array, since it can be updated in the container wrapper if required.
 		typeof minHeight === "number" && containerViewStyle.push({ minHeight });
-
+		// console.log(containerViewStyle, this.props.containerStyle);
```

Comment:
@pragadeeshk Do we need this commented line?

Reply:
Removed.